reconcile/utils/terrascript_aws_client.py

```diff
@@ -1181,6 +1185,64 @@ def populate_vpc_peerings(self, desired_state):
                     tf_resource = aws_route(route_identifier, **values)
                     self.add_resource(infra_account_name, tf_resource)
 
+    def populate_vpc_requests(self, vpc_requests: Iterable[VPCRequest]) -> None:
+        for request in vpc_requests:
+            # The default values here come from infra repo's module configuration
+            values = {
+                "source": "terraform-aws-modules/vpc/aws",
+                "version": "5.7.1",
+                "name": request.identifier,
+                "cidr": request.cidr_block.network_address,
+                "private_subnet_tags": {"kubernetes.io/role/internal-elb": "1"},
+                "public_subnet_tags": {"kubernetes.io/role/elb": "1"},
+                "create_database_subnet_group": False,
+                "enable_dns_hostnames": True,
+                "tags": {
+                    "Terraform": True,
```

Comment:
can this be set tothe usual tags?
- DEFAULT_TAGS (maybe not?)
- managed_by_integration can be very useful to tell that a resource is managed

Reply:
Yes. Do you happen to know the source for the usual tags?

Reply:
No definite source other than the definition of DEFAULT_TAGS in this very file and some managed_by_integration definitions scattered around. There is no standard as far as I can tell

Reply:
I see that DEFAULT_TAGS is used in the constructor for the class and get applied to resources without tags. Other methods here set `managed_by_integration` to `self.integration`. That worked for this one.

Does that solves the issue here?

Reply:
SGTM!